cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -67,8 +68,25 @@
     private boolean helpFlag;
 
     private static boolean isDirEmpty(final Path directory) throws IOException {
+
         try (DirectoryStream<Path> dirStream = Files.newDirectoryStream(directory)) {
-            return !dirStream.iterator().hasNext();
+            //Check whether the OS is MacOS and the folder contains .DS_Store file
+            Iterator pathIterator = dirStream.iterator();
+            if (pathIterator.hasNext()) {
```

Comment:
We can fail fast here by returning 'true' if there are no files inside the directory?

```java
if (!pathIterator.hasNext()) { 
 return true;
}
```